docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+```
+
+This file defines the additional links for adding a RealSense camera to the robot, as well as
+configuring the `openni_kinect` plugin for Gazebo to simulate data from a depth camera.  The
+camera itself will be connected to the Husky's `top_plate_front_link` link.  This places the
+camera at the very front edge of the robot's top cover-plate.
+
+Now, set the `HUSKY_URDF_EXTRAS` environment variable and try viewing the Husky model:
+
+```bash
+export HUSKY_URDF_EXTRAS=$HOME/Desktop/realsense.urdf.xacro
+roslaunch husky_viz view_model.launch
+```
+
+You should see the Husky model in RViz, with the RealSense camera mounted to it:
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/husky_images/husky_realsense.png"
+      width="800"
+    />
+    <figcaption>Husky with a RealSense D435 connected to it</figcaption>
+  </figure>
+</center>
+
+To launch the customized Husky in any of the new simulation environments, similarly run:
+
+```bash
+export HUSKY_URDF_EXTRAS=$HOME/Desktop/realsense.urdf.xacro
+roslaunch cpr_office_gazebo office_world.launch platform:=husky
+```
+
+You should see Husky spawn in the office world with the RealSense camera:
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/husky_images/husky_office_realsense.png"
+      width="800"
+    />
+    <figcaption>Husky with a RealSense D435 connected to it in Office World</figcaption>
+  </figure>
+</center>
+
+You can view the sensor data from the RealSense camera by running
+
+```bash
+roslaunch husky_viz view_robot.launch
+```
+
+and adding the camera and pointcloud from the `/realsense/color/image_raw` and
+`/realsense/depth/color/points` topics:
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/husky_images/husky_rviz_realsense.png"
+      width="800"
+    />
+    <figcaption>Husky with a RealSense in RViz showing pointcloud and RGB topics</figcaption>
+  </figure>
+</center>
+
+### Interfacing with Husky
+
+Both simulated and real Husky robots expose the same ROS interface, and can be interacted with 
+in the same way.
+
+:::note
+Please make sure that the desktop packages for Husky are installed:
+
+```bash
+sudo apt-get install ros-noetic-husky-desktop
+```
+:::
+
+#### Visualization
+
+You may start a preconfigured [RViz](http://wiki.ros.org/rviz) instance using the following command:
+
+```bash
+roslaunch husky_viz view_robot.launch
+```
+
+#### Control
+
+There are three ways to send your Husky control commands:
+
+1. Using the provided PS4 or Logitech controller. Refer to the [User Manual](/docs/robots/husky/user_manual_husky/#controller)
+   details on how to use the controller.
+
+2. Using the RViz instance above. If you select the Interact option in the top toolbar, an interactive
+   marker will appear around the Husky and can be used to control speed.
+
+3. The [rqt_robot_steering plugin](http://wiki.ros.org/rqt_robot_steering).
+   Run the `rqt` command, and select **Plugins->Robot Tools->Robot Steering** from the top menu.
```

Comment:
Consider using the ACSII arrow, →
This symbol is used on Docusaurus's site:

- https://github.com/facebook/docusaurus/blob/292d728c49182cdf063c73d3dd6f7a016a3f75ec/website/docs/guides/creating-pages.md?plain=1#L96
- https://docusaurus.io/docs/creating-pages

Reply:
Updated throughout